app/models/katello/content_view_docker_filter.rb

```diff
@@ -0,0 +1,34 @@
+module Katello
+  class ContentViewDockerFilter < ContentViewFilter
+    CONTENT_TYPE = 'docker'.freeze
```

Comment:
I had that originally but the user would see the filter not as "docker" but as "docker_tag" (iirc) which didn't appeal to me.